broker/src/main/java/org/apache/rocketmq/broker/processor/SendMessageProcessor.java

```diff
@@ -635,8 +636,16 @@ public SocketAddress getStoreHost() {
     }
 
     private String diskUtil() {
-        String storePathPhysic = this.brokerController.getMessageStoreConfig().getStorePathCommitLog();
-        double physicRatio = UtilAll.getDiskPartitionSpaceUsedPercent(storePathPhysic);
+        double physicRatio = -1;
+        String storePath =this.brokerController.getMessageStoreConfig().getStorePathCommitLog();
+        if (storePath.contains(MessageStoreConfig.MULTI_PATH_SPLITTER)) {
+            String[] paths = storePath.trim().split(MessageStoreConfig.MULTI_PATH_SPLITTER);
+            for (String storePathPhysic : paths) {
+                physicRatio = Math.min(physicRatio, UtilAll.getDiskPartitionSpaceUsedPercent(storePathPhysic));
```

Comment:
Is `Math.min()` correct here? `physicRatio` is initialized as `-1` and `UtilAll.getDiskPartitionSpaceUsedPercent()` always returns value greater or equal to `-1`.

Reply:
@caigy Yes, you are right. Fixed.